rundev

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
 # Privileged is needed for being able to inspect packages in the host system
-docker run -it --rm --net=host --privileged -v $(pwd)/config.json:/config.json -v ${PWD}/files/opt/dash-button/scripts:/opt/dash-button/scripts dev/dash-buttons
+docker run -it --rm --net=host --privileged -v $(pwd)/config:/opt/dash-button/config -v ${PWD}/files/opt/dash-button/scripts:/opt/dash-button/scripts dev/dash-buttons
```

Comment:
If I remember correctly @rponudic  said that we should place the config into a separate folder and we mount the hole directory so that a provider can write a configuration, if needed. The config is written back because we thought it would be easier to transfer, but we could also print the credentials into the console.